public/js/index.js

```diff
@@ -1458,29 +1458,26 @@ $(document).ready(function () {
     });
 
     // Helpers.
-    $('.delete-button').click(function () {
-        var $this = $(this);
-        var filter = "";
-        if ($this.attr("id")) {
-          filter += "#"+$this.attr("id")
+    $('.delete-button').click(showDeletePopup);
+
+    var prev_val = "{{branch_name}}";
```

Comment:
@Bwko I changed my code as @ethantkoenig suggested and in your code it would also be better to just generate `<span class="branch-name"></span>` in template and just use jquery to set value to that span using `.text($this.data('val'))` and you wont need to change modals html at all